cypress/integration/html-reader/page-button.ts

```diff
@@ -119,4 +138,42 @@ describe('PageButton visibility on useHtmlReader', () => {
       'not.be.disabled'
     );
   });
+
+  it('Scrolling mode & screen resize should show/hide the page buttons', () => {
+    cy.intercept('GET', 'https://alice.dita.digital/text/uncopyright.xhtml').as(
+      'uncopyright'
+    );
+
+    cy.findByRole('button', { name: 'Settings' }).click();
+    cy.findByText('Scrolling').click();
+
+    cy.findByRole('button', { name: 'Table of Contents' }).click();
+    cy.log('move to last chapter');
+    cy.findByRole('menuitem', { name: /(.*copy.*right$)/ }).click();
+
+    cy.wait('@uncopyright', { timeout: 10000 });
+
+    cy.wait(1000);
+
+    cy.getIframeBody().find('.copyright-page').should('exist');
+
+    cy.wait(1000);
+
+    cy.log('scroll to the bottom of the page');
+    cy.window().scrollTo('bottom');
+    cy.wait(1000);
+
+    cy.findByRole('button', { name: 'Next Page' }).should('be.disabled');
+    cy.findByRole('button', { name: 'Previous Page' }).should(
+      'not.be.disabled'
+    );
+
+    cy.log('Small screen should disable next button');
+    cy.viewport(100, 100);
+
+    cy.findByRole('button', { name: 'Next Page' }).should('be.disabled');
+    cy.findByRole('button', { name: 'Previous Page' }).should(
+      'not.be.disabled'
+    );
```

Comment:
I don't think this is testing what we want. We want to test that screen resize will trigger an update on the disabled state of the prev/next buttons. In this case, they stay the same.

Reply:
Hmm okay, this test is just not correct actually, next page button should always be disabled on scrolling mode cuz it's on the last page(For the current webreader version, it's chapter navigation). This is only true for paginated mode where a resize will trigger the next/previous state, we already have a test for that.  

Butttt, I think this is different in the new HTML renderer? I can make a comment to change this after the merge if this is what we decided to do.

Reply:
I think the new renderer and the old will function the same. When in scrolling mode, the next/prev button should take you one page unit down the page in scrolling mode (as well as paginated mode)

Reply:
I think we might have found a bug! It's chapter navigation for me on the current version.

Reply:
Ticket: [OE-376](https://jira.nypl.org/browse/OE-376)